contracts/contracts/strategies/FluxStrategy.sol

```diff
@@ -0,0 +1,24 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+/**
+ * @title Flux Strategy
+ * @notice Investment strategy for investing stablecoins via Flux
+ * @author Origin Protocol Inc
+ */
+
+import { CompoundStrategy } from "./CompoundStrategy.sol";
+
+contract FluxStrategy is CompoundStrategy {
+    constructor(BaseStrategyConfig memory _stratConfig)
+        CompoundStrategy(_stratConfig)
+    {}
+
+    /**
+     * @inheritdoc CompoundStrategy
+     */
+    function collectRewardTokens() external override {
```

Comment:
Let's just have the modifiers and pay the price. I'd like for us to be able to automatically test that the general public will get a permission denied response if trying to call any state changing method on our strategies.